ui/raidboss/data/07-dt/raid/r1s.ts

```diff
@@ -352,16 +214,16 @@ const triggerSet: TriggerSet<Data> = {
       },
     },
     {
-      id: 'R1S Headmarker Line Stack',
+      id: 'R1S Headmarker Nailchipper Spread',
       type: 'HeadMarker',
-      netRegex: { id: headMarkerData.lineStack, capture: true },
+      netRegex: { id: headMarkerData.spreadMarker1, capture: false },
       suppressSeconds: 5,
-      response: Responses.stackMarkerOn(),
+      response: Responses.spread(),
```

Comment:
Would it make sense for this to be `Out + Spread`, or something like that?  IIRC, whichever role gets the first Nailchippers first cannot bait the Quadruple Crossing; and the second set just goes off with/after the final cleave (that everyone is avoiding), so perhaps a reminder to be 'out' would be helpful?  Or perhaps 'spread' sufficiently implies that...

Reply:
Before I saw this PR, I made a similar change at https://github.com/OverlayPlugin/cactbot/pull/317

I think an out + spread would make the most sense.